controllers/managedfusionoffering_controller.go

```diff
@@ -85,13 +97,118 @@ func pluginReconcile(r *ManagedFusionOfferingReconciler) (ctrl.Result, error) {
 	if _, found := r.managedFusionOffering.Spec.Config["onboardingValidationKey"]; !found {
 		return ctrl.Result{}, fmt.Errorf("%s CR does not contain onboardingValidationKey entry", r.managedFusionOffering.Name)
 	}
+	if err := reconcileStorageCluster(r); err != nil {
+		return ctrl.Result{}, err
+	}
+
 	return ctrl.Result{}, nil
 }
 
+func reconcileStorageCluster(r *ManagedFusionOfferingReconciler) error {
+	r.Log.Info("Reconciling StorageCluster")
+	storageCluster := &ocsv1.StorageCluster{}
+	storageCluster.Name = "ocs-storagecluster"
+	storageCluster.Namespace = r.managedFusionOffering.Namespace
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, storageCluster, func() error {
+		if err := r.own(storageCluster); err != nil {
+			return err
+		}
+		sizeAsString := r.managedFusionOffering.Spec.Config["usableCapacityInTiB"]
+
+		// Setting hardcoded value here to force no MCG deployment
+		enableMCGAsString := "false"
```

Comment:
WE don't need MCG here

Reply:
curious why we don't need it here 🤔

Reply:
I think there is no harm in keeping MCG, anyway we are not enabling it.